src/components/Book.jsx

```diff
@@ -13,13 +14,23 @@ const Book = ({
   };
 
   return (
-    <li key={key}>
-      {title}
-      {' by '}
-      {author}
-      <button type="submit" onClick={handleClick}>
-        Remove
-      </button>
+    <li key={key} className="Book-card">
+      <div className="Book-detail">
+        <p className="Book-category">Action</p>
+        <p className="Book-title">{title}</p>
+        <p className="Book-author">{author}</p>
+        <div className="">
+          <button type="submit" className="Book-button">
+            Comment
+          </button>
+          <button type="submit" onClick={handleClick} className="Book-button">
+            Remove
+          </button>
+          <button type="submit" className="Book-button">
+            Edit
+          </button>
+        </div>
+      </div>
```

Comment:
- [ ] Some elements of the given design are missing. Kindly fix this.
`Original design`
![Screenshot from 2023-04-08 17-34-53](https://user-images.githubusercontent.com/65860678/230730202-0e41796a-38c7-4bbe-82d1-69c097061f6a.png)

`Your design`
![Screenshot from 2023-04-08 17-35-52](https://user-images.githubusercontent.com/65860678/230730286-20d62d24-fa1d-4adb-814b-997e33b9ecca.png)